python/cudf/__init__.py

```diff
@@ -6,8 +6,8 @@
 from cudf.dataframe import Index
 from cudf.dataframe import Series
 from cudf.multi import concat
-from cudf.io import (read_csv, read_parquet, read_feather, read_json,
-                     read_hdf, read_orc, from_dlpack)
+from cudf.io import (read_csv, read_parquet, read_parquet_metadata,
```

Comment:
I think it would be okay to reach into `cudf.io` to get `read_parquet_metadata`, this won't typically be used by end users.

Reply:
Makes sense. Reverted!